console/console-init/ui/src/components/About/About.tsx

```diff
@@ -59,7 +59,9 @@ export const About: React.FunctionComponent<IAboutProps> = ({
   if (docs) {
     docsElement = (
       <>
-        <TextListItem component="dt">Documentation</TextListItem>
+        <TextListItem component="dt" id="about-documentation-textlistitem">
```

Comment:
I think in the UI it looks like a dropdown, so it should be better named as 'about-documentation-dropdown'.